pkg/controller/clusterinstallation/utils.go

```diff
@@ -53,11 +53,52 @@ func (r *ReconcileClusterInstallation) checkClusterInstallation(mattermost *matt
 		return nil, fmt.Errorf("found %d pods, but wanted %d", len(pods.Items), mattermost.Spec.Replicas)
 	}
 
+	var mmEndpoint string
+	if mattermost.Spec.UseServiceLoadBalancer {
+		svc := &corev1.ServiceList{
+			TypeMeta: metav1.TypeMeta{
+				Kind:       "Service",
+				APIVersion: "v1",
+			},
+		}
+		err := r.client.List(context.TODO(), opts, svc)
+		if err != nil {
+			return nil, err
+		}
+		if len(svc.Items) > 1 {
+			return nil, fmt.Errorf("should return just one service but returned %d", len(svc.Items))
+		}
+		if svc.Items[0].Status.LoadBalancer.Ingress == nil {
+			return nil, fmt.Errorf("waiting for the Load Balancers be active")
+		}
+		if svc.Items[0].Status.LoadBalancer.Ingress[0].Hostname != "" {
```

Comment:
Do we care if the Hostname and the IP values are blank? Should we fall back to something else in that case?

Reply:
maybe `not available` ? @gabrieljackson

Reply:
That works for me. You can set `mmEndpoint := "not available"` when you initialize the variable above and then if it doesn't get overridden that will be the default.

Reply:
done @gabrieljackson